### DISCUSSION TITLE
Anyone have an example of using @nuxt/tailwindcss v.5.0.0-2

### DISCUSSION THREAD

**Discussion by procrates** [nuxt/nuxt#17927] [Questions] [answered]
I saw yesterday that a new Nuxt tailwind version was released. Although in beta it should work with Nuxt 3. Anyone has an example? 

Also what is the benefit of using this module instead of just installing tailwind 3 directly as per tailwind nuxt documentations?

Source: [Nuxt tailwind v.5](https://tailwindcss.nuxtjs.org/releases#Nuxt%203%20and%20Tailwindcss%203%20support)

**Comment by Diizzayy** [comment, marked as answer]
@procrates The nuxt tailwind github repo provides a playground which basically serves as an example. [Nuxt3 Tailwind Example](https://github.com/nuxt-community/tailwindcss-module/tree/main/playground)

As for the benefits of using the module as opposed to installing tailwind directly. The `@nuxt/tailwindcss` module is ready-to-go out the box, it eliminates you having to manually follow all the steps required to properly install and utilize tailwind. It follows best practices, provides a default tailwind config as well as allows you to utilize [Tailwind Config Viewer Tool](https://github.com/rogden/tailwind-config-viewer). All of which would definitely take much longer if you had to setup yourself
